src/containers/Tenant/Diagnostics/TopShards/TopShards.tsx

```diff
@@ -231,7 +257,7 @@ export const OverloadedShards = ({tenantPath, type}: OverloadedShardsProps) => {
             {
                 name: tableColumnsNames.IntervalEnd,
                 render: ({value}) => formatDateTime(new Date(value as string).getTime()),
```

Comment:
If it is `immediate` table, this fields (`IntervalEnd` and `PeakTime`) are missing. I think here it's needed to add some placeholders like dashes or somehow else say, that these fields are always empty for this mode

Reply:
i thought i addressed it
thanks
will probably remove these columns entirely for the immediate mode

Reply:
fixed